src/components/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Banner from "../Banner";
+import ImageWithStickers from "../ImageWithStickers";
+
+export interface PageHeaderProps {
+    title: string;
+    subtitle?: string;
+    image?: string;
+    removeTopSticker?: boolean;
+    topSticker?: string;
+    topStickerColor?: string;
+    bottomSticker?: string;
+    bottomStickerColor?: string;
+}
+
+const PageHeader = (props: PageHeaderProps) => {
+    return (
+        <div className="page-header__banner">
+            <Banner title={props.title} subtitle={props.subtitle} />
+            <div className="page-header__image-with-stickers" >
```

Comment:
Do you need the div "page-header__image-with-stickers"? Could you remove it if it's not necessary?